docs/integration-guides/key-management.md

```diff
@@ -57,25 +57,12 @@ For larger, more robust systems, external private key management is recommended.
 ---
 ### External accounting systems
```

Comment:
What do you think of adding a note/tip/info/quote here with the 2 most important bullet points for external systems - always confirm blocks, and guarantee idempotency.